common/ledger/blkstorage/fsblkstorage/blockfile_mgr_test.go

```diff
@@ -209,7 +209,7 @@ func TestBlockfileMgrGetTxByIdDuplicateTxid(t *testing.T) {
 		},
 		[]string{"txid-1", "txid-2", "txid-1"},
 	)
-	txValidationFlags := ledgerutil.NewTxValidationFlags(3)
+	txValidationFlags := txflags.NewTxValidationFlags(3)
```

Comment:
After moving this here, this functions (and the other one) can be renamed to remove redundant info in the package name and the function name. For instance, this function could simply be renamed as "New".

Reply:
yes, I agree. Will do.

Reply:
The following changes are done. 
`TxValidationFlags` -> `ValidationFlags`
`NewTxValidationFlags()` -> `New()`
`NewTxValidationFlagsSetValue()` -> `NewWithValues()`

Reply:
> The following changes are done.
> `TxValidationFlags` -> `ValidationFlags`
> `NewTxValidationFlags()` -> `New()`
> `NewTxValidationFlagsSetValue()` -> `NewWithValues()`

Looks good. I guess that "TxValidationFlags" could be changed to "Values" to match others renames. But I am fine merging this as the larger objective is achieved.

Reply:
If we do that, wouldn't API become little ugly?  -- `Values` vs `NewWithValues()` 
`TxValidationFlags` -> `Values`
`NewTxValidationFlags()` -> `New()`
`NewTxValidationFlagsSetValue()` -> `NewWithValues()`

The reasoning behind the current naming is that t`xflags.ValidationFlags` denotes that we have `txflags` and one type of the flag is validation flags. I know that we will not have any more types but this makes code more readable IMO.

Reply:
I have contradicted with myself there. With that reasoning, even the `New()` can be made `NewValidationFlags()`. Let's see what Gari suggests.

Reply:
As I had mentioned earlier, I am fine with any consistent naming. One alternate could be to use `txvalidationflags` for the package name, so we don't miss `validation` in the name and at the same time avoid repetition of `flags` in package name and in the name of functions/types.

Reply:
Let's go with what we have now. Don't want to get blocked on this.